pkg/config/config.go

```diff
@@ -201,6 +196,36 @@ func (c *ConfigInfo) SetActiveApplication(application string, project string) er
 	return nil
 }
 
+// AddApplication add  new application to the config file
+// Newly create application is NOT going to be se as Active.
+func (c *ConfigInfo) AddApplication(application string, project string) error {
+	if c.ActiveApplications == nil {
+		c.ActiveApplications = []ApplicationInfo{}
+	}
+
+	for i, app := range c.ActiveApplications {
```

Comment:
this loop sets the application as active. we already have `SetActiveApplication()`, should we have this here?

Reply:
Thanks for catching this.
Setting application as active shouldn't be there. This loop is there only to error out if the application already exists.

fixed